templates/conf.d/nginx.conf.erb

```diff
@@ -110,6 +110,12 @@ http {
   etag <%= @etag %>;
 
 <% end -%>
+<% if @gzip_static == 'on' -%>
```

Comment:
The alternative is to use:
```erb
<% if @gzip_static != 'off' -%>
   gzip_static       <%= @gzip_static %>;
<% end -%>
```

Reply:
Or should it be `Optional` as the [ngx_http_gzip_static module](https://nginx.org/en/docs/http/ngx_http_gzip_static_module.html) might not be available?

> This module is not built by default, it should be enabled with the --with-http_gzip_static_module configuration parameter.

([`ngx_http_gzip_module`](http://nginx.org/en/docs/http/ngx_http_gzip_module.html) is built by default and provides the options `gzip_vary`, `gzip_types` and `gzip_proxied`)

Reply:
> Or should it be `Optional` as the [ngx_http_gzip_static module](https://nginx.org/en/docs/http/ngx_http_gzip_static_module.html) might not be available?
> 
> > This module is not built by default, it should be enabled with the --with-http_gzip_static_module configuration parameter.
> 
> ([`ngx_http_gzip_module`](http://nginx.org/en/docs/http/ngx_http_gzip_module.html) is built by default and provides the options `gzip_vary`, `gzip_types` and `gzip_proxied`)

The default value is off, the static module is different to normal gzip module, the normal module compress "on the fly", and static serve pre-compressed files

Reply:
> The alternative is to use:
> 
> ```
> <% if @gzip_static != 'off' -%>
>    gzip_static       <%= @gzip_static %>;
> <% end -%>
> ```

change made